tests/vmi_multus_test.go

```diff
@@ -330,7 +330,24 @@ var _ = Describe("Multus", func() {
 					[]string{"sh", "-c", "ip a"},
 				)
 				Expect(err).ToNot(HaveOccurred())
-				Expect(strings.Contains(out, customMacAddress)).To(BeFalse())
+
+				// The mac address should exist on the pod if we are using the tuning cni plugin
```

Comment:
@rmohr I added this if because when we are one using tuning the mac on the pod will not be the same as the custom one.